### PR TITLE
[kuwo] fix title extraction and update test

### PR DESCRIPTION
**Pull request by remitamine**

### CODE REVIEW

File: youtube_dl/extractor/kuwo.py [hunk 1]
```diff
@@ -78,7 +79,7 @@ def _real_extract(self, url):
             errnote='Unable to get song detail info')
 
         song_name = self._html_search_regex(
-            r'<h1[^>]+title="([^"]+)">', webpage, 'song name')
+            r'(?s)class="(?:[^" ]+ +)*title(?: +[^" ]+)*".*?<h1[^>]+title="([^"]+)"', webpage, 'song name')
```
Comment (yan12125) [inline youtube_dl/extractor/kuwo.py]: According to the [W3C HTML5 syntax spec](http://www.w3.org/TR/html5/syntax.html), using ```'\s'``` to match whitespaces is better here.

**Comment by yan12125** [comment]
Could you add a check for the 'offline' error, too?

**Comment by remitamine** [comment]
i added the chack for offline error page and change match for whitespaces with `\s`

File: youtube_dl/extractor/kuwo.py [hunk 2]
```diff
@@ -76,9 +77,11 @@ def _real_extract(self, url):
         webpage = self._download_webpage(
             url, song_id, note='Download song detail info',
             errnote='Unable to get song detail info')
+        if '对不起，该歌曲由于版权问题已被下线，将返回网站首页' in webpage:
+            raise ExtractorError('this song has been offline because of copyright issues')
```
Comment (yan12125) [inline youtube_dl/extractor/kuwo.py]: Please add ```expected=True``` here as it's not a youtube-dl's fault.

**Comment by remitamine** [comment]
i changed the offline error into an expected ExtractorError

**Comment by yan12125** [comment]
Thanks. Merged.

**Pull request merged**